### ISSUE TITLE
Make robo dependency optional

### ISSUE THREAD

**Issue by sagikazarmark** [indigophp/robo-iless#2]
It is required for development, but not required if robo executable is in the PATH or installed globally. How about making it a suggestion?

**Comment by TamasBarta** [comment]
I thought about that when adding as requirement, but that eased development. Before making a release as per #1, this should be done.

**Comment by sagikazarmark** [comment]
As said, you can make it a dependency for development and suggested otherwise.

**Comment by TamasBarta** [comment]
Yes, but NOT TODAY :D I'll put time in it, but then I'll also want to polish it to a release level. This may take place on the weekend, or next week, since this week is extremely busy.

**Comment by sagikazarmark** [comment]
Did not mention when...
